server/channels/app/file_info.go

```diff
@@ -40,7 +41,9 @@ func getInfoForBytes(name string, data io.ReadSeeker, size int) (*model.FileInfo
 
 			if info.MimeType == "image/gif" {
 				// Just show the gif itself instead of a preview image for animated gifs
-				data.Seek(0, io.SeekStart)
+				if _, err := data.Seek(0, io.SeekStart); err != nil {
+					log.Printf("Failed to seek to the beginning of the data: %v", err)
```

Comment:
Let's not log to stdout and instead return an `*model.AppError`. Maybe there is an error code that you can reuse.